config.yaml

```diff
@@ -745,3 +745,45 @@ guilds:
     type: "Complete"
     encounterName: "The Omega Protocol (Ultimate)"
     to: "Alpha"
+- name: Deep Dungeon
+  roles:
+    relevantParsing: false
+    relevantFlexing: false
+    relevantRepetition: false
+    legend: false
+    ultimateFlexing: false
+    ultimateRepetition: false
+    datacenter: true
+  guildId: 1021602454716022845
+  channelId: 1172348608671133760
+  achievements:
+    - name: "Pal-less Palace III"
+      roles:
+        - name: "Pal-less Palace"
+          type: "Palace of the Dead"
```

Comment:
thoughts on having the specific deep dungeon `type`s vs having an overall "Deep Dungeon" `type`?

Reply:
also kinda iffy on how to put in the title achievement. wasn't really sure how to get the external name from the file and store it in config

Reply:
I think this makes a lot of sense for now.